flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/resources/ddl/column_type_test_mysql8.sql

```diff
@@ -0,0 +1,62 @@
+-- Copyright 2023 Ververica Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--   http://www.apache.org/licenses/LICENSE-2.0
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+
+-- ----------------------------------------------------------------------------------------------------------------
+-- DATABASE:  column_type_test
+-- ----------------------------------------------------------------------------------------------------------------
+
+CREATE TABLE full_types
+(
+    id                   SERIAL,
+    tiny_c               TINYINT,
+    tiny_un_c            TINYINT UNSIGNED,
+    tiny_un_z_c          TINYINT UNSIGNED ZEROFILL,
+    small_c              SMALLINT,
+    small_un_c           SMALLINT UNSIGNED,
+    small_un_z_c         SMALLINT UNSIGNED ZEROFILL,
+    medium_c             MEDIUMINT,
+    medium_un_c          MEDIUMINT UNSIGNED,
+    medium_un_z_c        MEDIUMINT UNSIGNED ZEROFILL,
+    int_c                INTEGER,
+    int_un_c             INTEGER UNSIGNED,
+    int_un_z_c           INTEGER UNSIGNED ZEROFILL,
+    int11_c              INT(11),
+    big_c                BIGINT,
+    big_un_c             BIGINT UNSIGNED,
+    big_un_z_c           BIGINT UNSIGNED ZEROFILL,
+    varchar_c            VARCHAR(255),
+    char_c               CHAR(3),
+    real_c               REAL,
+    float_c              FLOAT,
+    float_un_c           FLOAT UNSIGNED,
+    float_un_z_c         FLOAT UNSIGNED ZEROFILL,
+    double_c             DOUBLE,
+    double_un_c          DOUBLE UNSIGNED,
+    double_un_z_c        DOUBLE UNSIGNED ZEROFILL,
+    decimal_c            DECIMAL(8, 4),
+    decimal_un_c         DECIMAL(8, 4) UNSIGNED,
+    decimal_un_z_c       DECIMAL(8, 4) UNSIGNED ZEROFILL,
+    numeric_c            NUMERIC(6, 0),
+    big_decimal_c        DECIMAL(65, 1),
+    bit1_c               BIT,
+    tiny1_c              TINYINT(1),
+    boolean_c            BOOLEAN,
+    PRIMARY KEY (id)
+) DEFAULT CHARSET=utf8;
```

Comment:
Could we refer[1] to cover more types?
[1] https://github.com/ververica/flink-cdc-connectors/blob/4843ab2432fe77d3a019f8002ec765cf78e42a88/flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/resources/ddl/column_type_test.sql#L55